client/cloud_credentials.go

```diff
@@ -75,8 +75,9 @@ type GcpCredentialsCreatePayload struct {
 }
 
 type GcpCredentialsValuePayload struct {
-	ProjectId         string `json:"projectId"`
-	ServiceAccountKey string `json:"serviceAccountKey"`
+	ProjectId                          string `json:"projectId,omitempty"`
```

Comment:
added omitempty, because not all fields should be passed (depends on the type of request).

Reply:
so in this case if `projectId` is empty it will omit that field entirely?

Reply:
and the same goes for the two below?

Reply:
correct. Added it to them as well.
This will align with the API behavior.